src/app/stories/user-profile/user-profile.component.html

```diff
@@ -1,3 +1,48 @@
-<p>
-  user-profile works!
-</p>
+<div class="small-container">
+  <div class="inner-container">
+    <div class="form-block">
+      <h2 class="">Edit your profile</h2>
+      <form>
+        <label for="first-name">First name</label>
+        <input
+          type="text"
+          id="first-name"
+          placeholder="Enter your first name"
+        />
+        <label for="last-name">Last name</label>
+        <input type="text" id="last-name" placeholder="Enter your last name" />
+        <label for="email">Email</label>
+        <input type="text" id="email" placeholder="Enter your email" />
+      </form>
+    </div>
+    <div class="avatar-block">
+      <div class="image-preview">
+        <label for="fileAvatar">
+          <img
+            *ngIf="!avatarPreview"
```

Comment:
You can try `*ngIf"=!avatatarPreview; else avatarPreviewContainer"`
instead of `*ngIf="!avatarPreview" `and `*ngIf="avatarPreview"`

```
<ng-container #avatarPreviewContainer>
</..>
```